### PR TITLE
Add ingest2parquet step to superpipeline.

### PR DESCRIPTION
**Pull request by Mohammad-nassar10**

### CODE REVIEW

File: kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py [hunk 1]
```diff
@@ -51,111 +54,126 @@ def sample_code_ray_orchestrator(
     p2_pipeline_runtime_actor_options: str = '{"num_cpus": 0.8}',
     p2_pipeline_data_max_files: int = -1,
     p2_pipeline_data_num_samples: int = -1,
-    # Exact dedup step parameters
-    p3_name: str = "ededup",
+    # ingest to parquet step parameters
+    p3_name: str = "ingest2parquet",
```
Comment (daw3rd) [inline kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py]: The new transform name is code2parquet.  Merge from dev and you should get that.

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py (hunk 1 above)
Comment (Mohammad-nassar10) [reply kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py]: Done.

**Review by revit13 — APPROVED** [review]

File: kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py [hunk 2]
```diff
@@ -51,111 +54,126 @@ def sample_code_ray_orchestrator(
...
-    p3_ededup_n_samples: int = 10,
+    # code to parquet parameters
+    p3_code_to_parquet_supported_langs_file: str = "test/code_2_parquet/languages/lang_extensions.json",
+    p3_code_to_parquet_detect_programming_lang: bool = True,
+    p3_code_to_parquet_domain: str = "code",
+    ip3_ngest_to_parquet_snapshot: str = "github",
+    p3_code_to_parquet_s3_access_secret: str = "s3-secret",
     # overriding parameters
     p3_overriding_params: str = '{"ray_worker_options": {"image": "'
+    + code_to_parquet_image
```
Comment (roytman) [inline kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py:69]: strange indentation/new line separations, can you check it please

File: transforms/code/malware/ray/clamd.conf [hunk 3]
```diff
@@ -3,8 +3,8 @@
 #Please read /usr/share/doc/clamav-daemon/README.Debian.gz for details
 LocalSocket /var/run/clamav/clamd.ctl
 FixStaleSocket true
-LocalSocketGroup users
-LocalSocketMode 666
+#LocalSocketGroup users
```
Comment (roytman) [inline transforms/code/malware/ray/clamd.conf]: can we remove these commented-out lines?

File: transforms/code/malware/ray/Dockerfile [hunk 4]
```diff
@@ -49,8 +54,10 @@ COPY test-data/ test-data/
 ENV PYTHONPATH /home/ray
 
 # Put these at the end since they seem to upset the docker cache.
-ARG BUILD_DATE
-ARG GIT_COMMIT
 LABEL build-date=$BUILD_DATE
```
Comment (roytman) [inline transforms/code/malware/ray/Dockerfile]: can you put these Label lines at the end of the Dockerfile

File: kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py (hunk 2 above)
Comment (Mohammad-nassar10) [reply kfp/superworkflows/ray/kfp_v1/superworkflow_code_sample_wf.py:69]: It is a result of the "pre-commit" run.

File: transforms/code/malware/ray/clamd.conf (hunk 3 above)
Comment (Mohammad-nassar10) [reply transforms/code/malware/ray/clamd.conf]: Fixed.

File: transforms/code/malware/ray/Dockerfile (hunk 4 above)
Comment (Mohammad-nassar10) [reply transforms/code/malware/ray/Dockerfile]: Done.

**Review by roytman — APPROVED** [review]
LGTM

**Review by shivdeep-singh-ibm — APPROVED** [review]
Looks good to me. some else must approve too.

**Review by takuyagt — APPROVED** [review]
LGTM. Thank you!

**Review by daw3rd — APPROVED** [review]

**Pull request merged**